MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffractionPresenter.h

```diff
@@ -115,11 +115,10 @@ class MANTIDQT_CUSTOMINTERFACES_DLL EnggDiffractionPresenter
   void runConvetUnitsAlg(std::string workspaceName);
 
   void runCloneWorkspaceAlg(std::string inputWorkspace,
-	  std::string outputWorkspace);
+                            std::string outputWorkspace);
```

Comment:
Better to pass as const reference. I think this is an input, right? If so:
`const std::string &outputWorkspace` the it is explicit that it is input-only.